Templates/Android/generated/BaseEntityCollectionReferenceRequest.java.tt

```diff
@@ -39,37 +39,43 @@
                 .post(new<#=TypeName(c)#>, body);
     }
 
+<# if (c.GetFeatures().CanExpand) { #>
     /**
      * Sets the expand clause for the request
      *
      * @param value The expand clause
      * @return The updated request
      */
     public <#=ITypeCollectionReferenceRequest(c)#> expand(final String value) {
-        addQueryOption(new QueryOption("expand", value));
+        addQueryOption(new QueryOption("$expand", value));
```

Comment:
Yeah, it seemed to be - there are some other changes of this same variety contained in [this commit](https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/29/commits/ad9cb2596486cf72d8dd89f37404179944e75761)